portal/urls.py

```diff
@@ -19,5 +19,6 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('',include('user.urls')),
-    path('repo/', include('repo.urls'))
+    path('repo/', include('repo.urls')),
+    path('accounts/', include('django.contrib.auth.urls')),
```

Comment:
can you provide a screenshot of what `accounts/` looks like, and why you needed that?

Reply:
I used it for login feature:
![screenshot-127 0 0 1-8900-2019 07 02-13-59-50](https://user-images.githubusercontent.com/43813460/60497080-c2189680-9cd1-11e9-8cfa-e6ac8d4b4344.png)

Reply:
where is Github Login ?

Reply:
and why is is called `accounts/`? It should be `/login` right?